Dockerfile.dev

```diff
@@ -1,25 +1,40 @@
-FROM debian:9
+FROM python:3
+
 LABEL maintainer "gordon@leastauthority.com"
```

Comment:
The correct syntax for `LABEL` is `LABEL maintainer="gordon@leastauthority.com"`.  You are missing an `=`.

REF : https://docs.docker.com/engine/reference/builder/#label

_A cool way to add name and email also looks like `LABEL maintainer="Gordon <gordon@leastauthority.com>"`_

Reply:
I'm not very familiar with Docker, but the document @Fenn-CS has linked suggests the label `org.opencontainers.image.authors` to set maintainer label.  

Also, I'm unsure that the email `gordon@leastauthority.com` exists anymore.  Should/could @Derkades be the maintainer?

Reply:
Yeah I could update it with my own email, replace it with a generic `@tahoe-lafs.org` email if such a thing exists, or even remove it entirely (I don't think anyone uses the maintainer label anyway)

Reply:
Any of those should be fine. :)

Reply:
Would the maintainer field help human readers to decide where to direct questions? If that is the case, I would think that `tahoe-dev@lists.tahoe-lafs.org` (the mailing list) might be a little more appropriate than individual ownership.  

But then the term `authors` suggests to me that the field is about authorship. If that is the case, both @Derkades and `gordon@leastauthority.com` would be the right authors, even if the latter turns out to be be an address that is not currently valid.

If this is too trivial to belabor, I wouldn't oppose removing the thing. :-)